azkaban-common/src/main/java/azkaban/executor/container/ContainerCleanupManager.java

```diff
@@ -174,9 +174,21 @@ public ContainerCleanupManager(
             new Pair<>(Duration.ofMinutes(runningFlowValidity), START_TIME))
         .build();
 
-    this.executionStoppedMap = new Builder<Status, Pair<Duration, String>>()
+    this.recentTerminatedStatusMap = new Builder<Status, Pair<Duration, String>>()
         .put(Status.EXECUTION_STOPPED,
-            new Pair<>(Duration.ofMinutes(maxExecStoppedValidity), UPDATE_TIME))
+            new Pair<>(Duration.ofMinutes(recentTerminationValidity), UPDATE_TIME))
```

Comment:
What's the utility of the second field of the pair, all of them have the value as 'update_time' in the map ?

Reply:
This is to fit the method being called below, it takes in `status` and a map of `<status, <retention, timestamp_tupe>>`